swift/internal/compiling.bzl

```
@@ -774,6 +775,13 @@ def compile_action_configs(
             ],
             features = [SWIFT_FEATURE_ENABLE_SKIP_FUNCTION_BODIES],
         ),
+        # Note: this *must* go before the index_while_building configurator
+        # as runner code processes this in order.
+        swift_toolchain_config.action_config(
+            actions = [swift_action_names.COMPILE],
+            configurators = [_global_index_store_configurator],
+            features = [SWIFT_FEATURE_USE_GLOBAL_INDEX_STORE],
```

I assume we want this to also include `SWIFT_FEATURE_INDEX_WHILE_BUILDING`?
```suggestion
            features = [
                SWIFT_FEATURE_INDEX_WHILE_BUILDING,
                SWIFT_FEATURE_USE_GLOBAL_INDEX_STORE,
            ],
```

---

Sure, we can auto enable this. ATM it requires both